### ISSUE TITLE
(with patch) It should give a useful error if it finds a directory whose name ends in .feature

### ISSUE THREAD

**Issue by sleekweasel** [vsiakka/gherkin-lint#216]
I had a directory whose name ended in '.feature' because it appeared as a java module coverage directory.

The resulting stacktrace was very frustrating, not giving the filename in question:

```
  return binding.read(fd, buffer, offset, length, position);

Error: EISDIR: illegal operation on a directory, read
    at Object.fs.readSync (fs.js:675:18)
    at tryReadSync (fs.js:540:20)
    at Object.fs.readFileSync (fs.js:583:19)
    at /Users/tim.baverstock/git/bma-calabash/node_modules/gherkin-lint/dist/linter.js:16:24
    at Array.forEach (<anonymous>)
    at Object.lint (/Users/tim.baverstock/git/bma-calabash/node_modules/gherkin-lint/dist/linter.js:12:9)
    at Object.<anonymous> (/Users/tim.baverstock/git/bma-calabash/node_modules/gherkin-lint/dist/main.js:24:22)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
```

It was easy to wrap in linter.js:

```
 14     let fileContent;
 15     try {
 16       fileContent = fs.readFileSync(fileName, 'utf-8');
 17     } catch (e) {
 18       e.message = e.message + " (" + fileName + ")";
 19       throw e;
 20     }
```

I'm in a rush to get home; it was easier to give the patch here than fork.
